### PR TITLE
Fix alignment issue in zoom knob when uploading avatar

### PR DESCRIPTION
**Pull request by PankajAS**
### Details

In order to vertically center the Knob, I removed the top style property and set justifyContent: 'center'. As for horizontally centering the rotation button, I removed its height property to prevent overflow. 

### Fixed Issues

$ [#16091 ](https://github.com/Expensify/App/issues/16091)
PROPOSAL: [#16091 comment](https://github.com/Expensify/App/issues/16091#issuecomment-1482487643)

### Tests

1. Go to setting > Profile
2. click on profile pic and select any picture
3. check that knob is aligned in center with the slider, and slider, knob and rotate button are aligned inline

- [x] Verify that no errors appear in the JS console

### Offline tests

same as online

### QA Steps

Check that knob is aligned in center with the slider, and slider, knob and rotate button are aligned inline
1. Go to setting > Profile
2. click on profile pic and select any picture
3. check that knob is aligned in center with the slider, and slider, knob and rotate button are aligned inline

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

https://user-images.githubusercontent.com/17126628/231523669-dd268ef1-bd7b-4419-9841-5618fc595238.mp4

</details>

<details>
<summary>Mobile Web - Chrome</summary>

https://user-images.githubusercontent.com/17126628/231523871-572e4d1e-d214-4dfd-8fbf-bf774e87b6ee.mp4

</details>

<details>
<summary>Mobile Web - Safari</summary>

https://user-images.githubusercontent.com/17126628/231524363-3219a53c-8986-4aec-bf1b-4ff333ff1b76.mp4

</details>

<details>
<summary>Desktop</summary>

https://user-images.githubusercontent.com/17126628/231524125-d8a91d78-18a9-4891-9a2a-264dc5f6cd3b.mp4

</details>

<details>
<summary>iOS</summary>

https://user-images.githubusercontent.com/17126628/231524911-d03d02cb-1921-4b73-af5d-eeb81e19b008.mp4

</details>

<details>
<summary>Android</summary>

https://user-images.githubusercontent.com/17126628/231524299-a916301e-4b2a-425d-847e-f60f5c7cce1e.mp4

</details>

### CODE REVIEW

**Comment by Santhosh-Sellavel** [comment]
not sure what went wrong here @mountiny

**Review by Santhosh-Sellavel — COMMENTED** [review]
Adding myself as reviewer

**Comment by mountiny** [comment]
@PankajAS you have to link the issue with a full link 
image
 as the PR template suggests

**Comment by PankajAS** [comment]
> @PankajAS you have to link the issue with a full link image as the PR template suggests

Updated full link with issue number, check now @mountiny

**Comment by mountiny** [comment]
@PankajAS Thanks, this is for future reference, you have to have it right when the PR is ready for review, then it will assign the correct reviewers

**Review by MonilBhavsar — COMMENTED** [review]
Thanks for the PR!
Please fill the details, tests and offline tests section. You can mention same as online if it is not related to offline.
Also, I think there is no need to mention issue number in PR title and it can be more meaningful and relatable, For example: "Fix alignment issue in zoom knob when uploading avatar"

**Comment by PankajAS** [comment]
> Thanks for the PR! Please fill the details, tests and offline tests section. You can mention same as online if it is not related to offline. Also, I think there is no need to mention issue number in PR title and it can be more meaningful and relatable, For example: "Fix alignment issue in zoom knob when uploading avatar"

Updated @MonilBhavsar , please check

**Comment by PankajAS** [comment]
> Thanks! Please fill "Details" section and describe briefly what this PR is doing. And a small update in tests/QA
> 
> > check that knob is aligned in center with the slider, and slider, knob and rotate button are aligned inline

Updated @MonilBhavsar

**Review by bondydaa — DISMISSED** [review]

**Comment by PankajAS** [comment]
@MonilBhavsar we can merge pull request today

**Comment by Santhosh-Sellavel** [comment]
As this is a UI Improvement, @Expensify/design does this look good?

**Comment by Santhosh-Sellavel** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

Screenshot 2023-04-15 at 5 09 47 AM

</details>

<details>
<summary>Mobile Web - Chrome</summary>

mWeb_Screenshot_20230415_050652

</details>

<details>
<summary>Mobile Web - Safari</summary>

Simulator Screen Shot - iPhone 14 - 2023-04-15 at 05 06 45

</details>

<details>
<summary>Desktop</summary>

Screenshot_20230415_050452

</details>

<details>
<summary>iOS</summary>

Simulator Screen Shot - iPhone 14 - 2023-04-15 at 05 04 36

</details>

<details>
<summary>Android</summary>

Screenshot 2023-04-15 at 5 10 44 AM

</details>

File: src/styles/styles.js [hunk 1]
```diff
@@ -2841,6 +2840,7 @@ const styles = {
         height: variables.sliderBarHeight,
         borderRadius: variables.sliderBarHeight / 2,
         alignSelf: 'stretch',
+        justifyContent: 'center',
     },
 
     imageCropRotateButton: {
```
Comment (Santhosh-Sellavel) [inline src/styles/styles.js:2846]: I think this should be removed as it is no longer required.

Also should be removed from here

https://github.com/Expensify/App/blob/6ee5e7a5afa5c7066368be9cee1f6b80f4699389/src/components/AvatarCropModal/AvatarCropModal.js#L100

**Review by Santhosh-Sellavel — CHANGES_REQUESTED** [review]

File: src/styles/styles.js (hunk 1 above)
Comment (PankajAS) [reply src/styles/styles.js]: i have removed  imageCropRotateButton.height from relativeHeight @Santhosh-Sellavel, please check

**Comment by PankajAS** [comment]
@Santhosh-Sellavel i have updated changes, check now

**Comment by shawnborton** [comment]
Looks good to me!

**Review by Santhosh-Sellavel — APPROVED** [review]
Thanks, @shawnborton!

LGTM, as well!

cc: @bondydaa or @MonilBhavsar all you!

**Review by MonilBhavsar — APPROVED** [review]

File: src/components/AvatarCropModal/AvatarCropModal.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ const AvatarCropModal = (props) => {
         const {height, width} = event.nativeEvent.layout;
 
         // Even if the browser height is reduced too much, the relative height should not be negative
-        const relativeHeight = Math.max(height - styles.imageCropRotateButton.height, CONST.AVATAR_CROP_MODAL.INITIAL_SIZE);
+        const relativeHeight = Math.max(height, CONST.AVATAR_CROP_MODAL.INITIAL_SIZE);
```
Comment (shubham1206agra) [inline src/components/AvatarCropModal/AvatarCropModal.js:100]: This change caused a bug where the wrong overlap of the profile editor was present. More details - https://github.com/Expensify/App/issues/62015

**Pull request merged**